### PR TITLE
fix(labels): refactor labels templates - INNO-703

### PR DESCRIPTION
**Pull request by yhuard**
# PR description

Refactor the templates of labels and radios as an example. You can either pass `label` property or block to the new template 💪 

# QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have given the fractal status “ready” to my component
- [x] I have declared `@define mycomponent` in the SCSS file
- [x] I have specified `margin: 0;` on the CSS component
- [x] I have provided tests
- [x] I follow the naming guidelines
- [x] the component supports composition
- [x] the template is fully functional
- [x] I have filled the README.md file (at least a few lines)
- [x] I have checked the dependencies
- [x] there are no hardcoded strings (all content come from the context)

### CODE REVIEW

File: framework/components/ecl-forms/ecl-forms-labels/ecl-forms-labels.twig [hunk 1]
```diff
@@ -1 +1,21 @@
...
+    - "label" (string): label (mandatory)
+    - "for_attribute" (string): ID of the element (optional)
+    - "extra_classes" (string): extra CSS classes to be added
+    - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+#}
+
+{% set _cssClass = 'ecl-form-label' %}
+
+{% if extra_classes is defined %}
+  {% set _cssClass = _cssClass ~ ' ' ~ extra_classes %}
```
Comment (kalinchernev) [inline framework/components/ecl-forms/ecl-forms-labels/ecl-forms-labels.twig]: Why not `_css_class`?

**Review by kalinchernev — CHANGES_REQUESTED** [review]
You have missed 1 variable convention :P

**Comment by kalinchernev** [comment]
Honestly, I have a second thought of including a screenshot for the IE in Windows. It's obviously something not taken care of and does not have much value to have in regression tests.

File: framework/components/ecl-forms/ecl-forms-labels/ecl-forms-labels.twig (hunk 1 above)
Comment (yhuard) [inline framework/components/ecl-forms/ecl-forms-labels/ecl-forms-labels.twig]: My bad, I fix it

**Review by kalinchernev — DISMISSED** [review]

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**